### ISSUE TITLE
ci.swift.org: Failing test: Required.Proto3.JsonInput.RepeatedListValue.JsonOutput

### ISSUE THREAD

**Issue by harlanhaskins** [apple/swift-protobuf#853]
This has failed a couple of times on the bots:

https://ci.swift.org/job/swift-protobuf-linux-ubuntu-14_04/1012/console
https://ci.swift.org/job/swift-protobuf-linux-ubuntu-16_04/1067/console

```
18:30:42 CONFORMANCE TEST BEGIN ====================================
18:30:42 
18:30:42 ERROR, test=Required.Proto3.JsonInput.RepeatedListValue.ProtobufOutput: Failed to parse input or produce output. request=json_payload: "{\n        \"repeatedListValue\": [[\"a\"]]\n      }" requested_output_format: PROTOBUF message_type: "protobuf_test_messages.proto3.TestAllTypesProto3" test_category: JSON_TEST, response=parse_error: "JSON failed to parse: unknownField(\"repeatedListValue\")"
18:30:42 ERROR, test=Required.Proto3.JsonInput.RepeatedListValue.JsonOutput: Failed to parse input or produce output. request=json_payload: "{\n        \"repeatedListValue\": [[\"a\"]]\n      }" requested_output_format: JSON message_type: "protobuf_test_messages.proto3.TestAllTypesProto3" test_category: JSON_TEST, response=parse_error: "JSON failed to parse: unknownField(\"repeatedListValue\")"
18:30:42 
18:30:42 These tests failed.  If they can't be fixed right now, you can add them to the failure list so the overall suite can succeed.  Add them to the failure list by running:
18:30:42   ./update_failure_list.py Sources/Conformance/failure_list_swift.txt --add failing_tests.txt
18:30:42 
18:30:42   Required.Proto3.JsonInput.RepeatedListValue.JsonOutput
18:30:42   Required.Proto3.JsonInput.RepeatedListValue.ProtobufOutput
18:30:42 
18:30:42 CONFORMANCE SUITE FAILED: 837 successes, 0 skipped, 0 expected failures, 2 unexpected failures.
```

**Comment by tbkka** [comment]
Simple fix:  A new conformance test case uses a new field in the test proto.  After updating the test proto in our sources, the test now passes.

**Comment by thomasvl** [comment]
Fixed in #855
